ui-tests/src/test/java/org/openmrs/reference/ManageProviderSchedulesTest.java

```diff
@@ -39,7 +39,7 @@
 	@Before
 	public void setUp() throws Exception {
 		homePage = new HomePage(page);
-		assertPage(homePage);
+		assertPage(homePage.waitForPage());
```

Comment:
For `assertPage()`to be effective in checking the current page, we need to wait the `homePage` to load